src/main/java/Duke.java

```diff
@@ -1,10 +1,306 @@
+        }
+    }
+
+    private static void executeCommand(String command) throws DukeException {
+        StringTokenizer st = new StringTokenizer(command);
+        String identifier = st.nextToken().toLowerCase();
+        String arguments = "";
+        while(st.hasMoreTokens()) {
+            arguments += st.nextToken() + " ";
+        }
+        arguments = arguments.trim();
+        switch (identifier) {
+        case "bye":
+            exitProgram();
+            break;
+        case "list":
+            printTaskList(arguments);
+            break;
+        case "mark":
+            markTask(arguments);
+            break;
+        case "unmark":
+            unmarkTask(arguments);
+            break;
+        case "todo":
+            createToDoTask(arguments);
+            break;
+        case "deadline":
+            createDeadlineTask(arguments);
+            break;
+        case "event":
+            createEventTask(arguments);
+            break;
+        case "delete":
+            deleteTask(arguments);
+            break;
+        default:
+            System.out.println("なに？！");
+            break;
+        }
+        try {
+            if (!identifier.equals("list")) {
+                writeTaskListData();
+            }
+        } catch (IOException e) {
+            System.out.println("Error");
+        }
+        System.out.println("Total task count: " + tasks.size() + ".\n" + gap);
+    }
+
+    private static void exitProgram() {
+        System.out.println(gap + "Goodbye! Have a pleasant day!\n" + gap);
+        System.exit(0);
+    }
+
+    private static void printTaskList(String arguments) throws DukeException {
+        if (!arguments.isEmpty()) {
+            throw new DukeException("list command does not accept arguments.\n"
+                    + "Enter 'list' to view the current list of tasks");
+        }
+        String taskList = "";
+        for (int i = 0; i < tasks.size(); i++) {
+            Task curr = tasks.get(i);
+            taskList += ((i+1) + ". " + curr.toString() + "\n");
+        }
+        System.out.println(gap + "Here are your tasks:\n" + taskList + gap);
+    }
+
+    private static void markTask(String arguments) throws DukeException {
+        int taskNum = Integer.parseInt(arguments) - 1;
+        Task selectedTask = tasks.get(taskNum);
+        selectedTask.markDone();
+        System.out.println(gap + "Well done, task marked as completed.\n"
+                + selectedTask.toString() + "\n" + gap);
+    }
+
+    private static void unmarkTask(String arguments) throws DukeException {
+        int taskNum = Integer.parseInt(arguments) - 1;
+        Task selectedTask = tasks.get(taskNum);
+        selectedTask.markNotDone();
+        System.out.println(gap + "Understood, task marked as uncompleted.\n"
+                + selectedTask.toString() + "\n" + gap);
+    }
+
+    private static void createToDoTask(String arguments) throws DukeException {
```

Comment:
I think it is better to separate these operations into respective files/ classes. It feels very confusing when there are so many methods in one file